### DISCUSSION TITLE
Custom Avatar

### DISCUSSION THREAD

**Discussion by rj0217** [ChristopherKlay/StadiaEnhanced#153] [Ideas & Requests]
I understand the custom stadia avatar is linked to the stadia enhanced extension but is there a way to make the custom avatar show up in the mobile app or to everyone else? Kinda sucks if it's only visible to you :(

**Comment by ChristopherKlay** [comment]
The issue with having avatars being shared to other users of the extension, is that it would require some sort of central database that contains the list of custom set avatars.

Stadia Enhanced is currently pretty much build around self-containment to remove any monthly cost for the project. Setting up a service for a feature like this, would likely need to change that.
